example/lib/main.dart

```diff
@@ -47,686 +48,631 @@ class _DartVLCState extends State<DartVLC> {
         this.setState(() => this.general = general);
       });
       this.devices = await Devices.all;
-      this.setState(() {}); 
-      Record record = await Record.create(
-        id: 205, 
-        media: await Media.network(Uri.parse('http://22533.live.streamtheworld.com/LOS40.mp3')), 
-        pathFile: 'C:\\record_${DateTime.now().millisecondsSinceEpoch}.mp3'
-      );
+      this.setState(() {});
+      Record record = await Record.create(id: 205, media: await Media.network(Uri.parse('http://22533.live.streamtheworld.com/LOS40.mp3')), pathFile: 'C:\\record_${DateTime.now().millisecondsSinceEpoch}.mp3');
       record.start();
-      Future.delayed(Duration(seconds: 10), ()=> record.stop());
+      Future.delayed(Duration(seconds: 10), () => record.stop());
     }
   }
 
+  @override
+  void initState() {
+    super.initState();
+    initializePlayer = _initializePlayer();
+  }
+
+  Future<Player> _initializePlayer() async {
```

Comment:
Why are we creating another instance of `Player` when one already exists in `didChangeDependencies`?
I think previous approach is fine, until #3 is worked on.

Reply:
I tried using the `didChangeDependencies` way but I'm getting this error when passing the `player: player!` argument to the `Video` widget

```
════════ Exception caught by widgets library ═══════════════════════════════════
The following _CastError was thrown building DartVLC(dirty, state: _DartVLCState#7d263):
Null check operator used on a null value

The relevant error-causing widget was
DartVLC
package:dart_vlc_example/main.dart:7
When the exception was thrown, this was the stack
#0      _DartVLCState.build
package:dart_vlc_example/main.dart:75
#1      StatefulElement.build
package:flutter/…/widgets/framework.dart:4713
#2      ComponentElement.performRebuild
package:flutter/…/widgets/framework.dart:4596
#3      StatefulElement.performRebuild
package:flutter/…/widgets/framework.dart:4769
#4      Element.rebuild
package:flutter/…/widgets/framework.dart:4286
...

```